templates/supplier_app/taxpayer-creation.html

```diff
@@ -2,73 +2,24 @@
 
 {% block content %}
 {% include 'supplier_app/navbar.html' %}
-<form method="post">
+<form method="post" enctype="multipart/form-data">
     {% csrf_token %}
     <div class="row m-3">
         <div class="col-md-6 form-group">
             <h4>Taxpayer info</h4>
-            <div>
-                {{taxpayer_form.name.label}}
-                {{taxpayer_form.name}}
-            </div>
-            <div class="form-row">
-                <div class="col-md-4">
-                    {{taxpayer_form.razon_social.label}}
-                    {{taxpayer_form.razon_social}}
-                </div>
-                <div class="col-md-4">
-                    {{taxpayer_form.cuit.label}}
-                    {{taxpayer_form.cuit}}
-                </div>
-                <div class="col-md-4">
-                    {{taxpayer_form.forma_de_pago.label}}
-                    {{taxpayer_form.forma_de_pago}}
-                </div>
-            </div>
-            <div>
-                {{taxpayer_form.justificacion.label}}
-                {{taxpayer_form.justificacion}}
-            </div>
+            {{taxpayer_form}}
```

Comment:
If we are making this abstraction, I'd identify which fields are common to all forms, and which don't.

Reply:
I will consider that